nano/node/nodeconfig.cpp

```diff
@@ -246,6 +250,12 @@ bool nano::node_config::upgrade_json (unsigned version_a, nano::jsonconfig & jso
 			upgraded = true;
 		}
 		case 16:
+		{
+			json.put ("tcp_client_timeout", tcp_client_timeout.count ());
+			json.put ("tcp_server_timeout", tcp_server_timeout.count ());
+			upgraded = true;
```

Comment:
Should this have a test? I added one in a PR but it's not reviewed yet, feel free to copy and replace with your options: https://github.com/nanocurrency/nano-node/pull/1734/files

Reply:
Thanks